applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_solver.py

```diff
@@ -128,9 +128,14 @@ def GetDefaultParameters(cls):
             "gradient_dofs" : false,
             "line_search": false,
             "compute_reactions": true,
-            "block_builder": true,
+            "builder_and_solver_settings" : {
+                "use_block_builder" : true,
+                "use_lagrange_BS"   : false,
```

Comment:
```suggestion
                "use_lagrange_builder_and_solver"   : false,
```
to follow the convention. Indeed, we are already inside the B&S settings so I'd put
```suggestion
                "use_lagrange_multipliers"   : false,
```
to avoid being redundant.

Reply:
Then we should change it in the StructuralApplication

Reply:
what does it mean "use_lagrange_bs"?

Reply:
use_lagrange_builder_and_solver. I t is the key currently used in the StructuralMechanicsApplication

Reply:
i was referring to "what does that BS do?"

Reply:
It was introduced several years ago, it solves using LM for constraints instead of the Felippa algoritihm.